{"name": "test_case_3[--]", "status": "failed", "statusDetails": {"message": "AssertionError: assert '昵称 2~16 个字符之间' == '编辑成功'\n  - 编辑成功\n  + 昵称 2~16 个字符之间", "trace": "self = <test_personalsave.Test_PersonalSave object at 0x7fb972987be0>, nickname = '', gender = '', birthday = ''\n\n    @pytest.mark.parametrize('nickname,gender,birthday', [('', '', '')])\n    def test_case_3(self, nickname, gender, birthday):\n        data = {\n            'nickname': nickname,\n            'gender': gender,\n            'birthday': birthday\n        }\n        res = self.perobj.persave(data)\n        print(res)\n>       assert res['msg'] == '编辑成功'\nE       AssertionError: assert '昵称 2~16 个字符之间' == '编辑成功'\nE         - 编辑成功\nE         + 昵称 2~16 个字符之间\n\ntestcase/test_personalsave.py:43: AssertionError"}, "attachments": [{"name": "stdout", "source": "9b246cb3-8fbc-4c96-ba14-1c613d9db99f-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "nickname", "value": "''"}, {"name": "gender", "value": "''"}, {"name": "birthday", "value": "''"}], "start": 1616642854751, "stop": 1616642854887, "uuid": "031cb6f6-393d-4574-88b1-06b29d5c3af8", "historyId": "4a8546d82a83c463a15b72a3db6cc17f", "testCaseId": "fb131b1e311fb2d8187e313d2b154295", "fullName": "testcase.test_personalsave.Test_PersonalSave#test_case_3", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_personalsave"}, {"name": "subSuite", "value": "Test_PersonalSave"}, {"name": "host", "value": "chenjinfeideMacBook-Air.local"}, {"name": "thread", "value": "20081-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_personalsave"}]}